content/en/docs/external-add-ons/feature-store/getting-started.md

```diff
@@ -13,7 +13,7 @@ For an overview of Feast, please read [Introduction to Feast](/docs/external-add
 
 **Overview**
 
-* This guide assumes that you have a running Kubeflow cluster already. If you don't have Kubeflow installed, then head on over to the [Kubeflow installation guide](/docs/started/getting-started/).
+* This guide assumes that you have a running Kubeflow cluster already. If you don't have Kubeflow installed, then head on over to the [Kubeflow installation guide](/docs/started/getting-started).
```

Comment:
Also this change should be removed (its not in the pipelines section).

Reply:
Reverted.